src/add/observable/combineLatest.ts

```diff
@@ -1,6 +1,4 @@
 import {Observable} from '../../Observable';
 import {combineLatestStatic} from '../../operator/combineLatest';
 
-Observable.combineLatest = combineLatestStatic;
-
-export var _void: void;
```

Comment:
@kwonoj Yes, I ran into that issue. Discussing with the TypeScript folks in Microsoft/TypeScript#6022